ext/nnstreamer/extra/nnstreamer_grpc_flatbuf.cc

```diff
@@ -227,9 +227,6 @@ SyncServiceImplFlatbuf::start_client (std::string address)
 
   /* connect the server */
   client_stub_ = TensorService::NewStub (channel);
-  if (client_stub_.get () == nullptr)
```

Comment:
```
if (client_stub_ == nullptr || client_stub_.get () == nullptr)
```
?

Reply:
```
WID:13797032 The pointer returned by operator new can't be null, but it's compared with null at nnstreamer_grpc_flatbuf.cc:230.
  if (client_stub_ == nullptr || client_stub_.get () == nullptr)
```
The same defect is detected

Reply:
How about applying try-catch exception handling?

Reply:
Thank you for the suggestion!
Updated to use try-catch